packages/botonic-plugin-contentful/tests/search/search-by-keywords.test.ts

```diff
@@ -134,7 +134,7 @@ export function keywordsWithMockCms(
   stemBlackList: { [locale: string]: StemmingBlackList[] } = {}
 ): SearchByKeywords {
   const mockCms = mock(DummyCMS)
-  when(mockCms.contentsWithKeywords(deepEqual(context))).thenResolve(
+  when(mockCms.contentsWithKeywords(deepEqual(context), undefined)).thenResolve(
```

Comment:
Since it's optional, you don't need to pass `undefined`, you can ignore the passing

Reply:
I added the `undefined` because the test was failing.
I've changed it to `anything()` to keep it clearer, but the thing is that mockito doesn't accept the `when` configuration to be set without an optional parameter and then calling the mocked function with this optional parameter setted (and vice versa) because it won't return the desired value, it will return `null`.
As you can see, there's an [open issue](https://github.com/NagRock/ts-mockito/issues/196) regarding this situation, it seems that the expected behaviour when configuring the `when` is not 100% clear.

Reply:
Ah right, go for it then!